yarn-project/noir-protocol-circuits-types/src/index.test.ts

```diff
@@ -89,6 +80,7 @@ describe('Private kernel', () => {
   });
 });
 
+// TODO(benesjan): move the following to relevant class tests
```

Comment:
Should this one have an issue?

Reply:
Addressed the issue in some of the newer commits.